docs/DebuggingNumericalError.md

```diff
@@ -32,50 +32,36 @@ reference inputs and outputs in protobuf.
 
 ```bash
 $ python ../utils/RunONNXModel.py  --help
-usage: RunONNXModel.py [-h] [--print-input] [--print-output]
-                       [--save-onnx PATH] [--save-so PATH | --load-so PATH]
-                       [--save-data PATH]
+usage: RunONNXModel.py [-h] [--model MODEL] [--compile-args COMPILE_ARGS] [--compile-only] [--compile-using-input-shape]
+                       [--print-input] [--print-output] [--save-onnx PATH] [--save-data PATH] [--verify {onnxruntime,ref}]
+                       [--verify-all-ops] [--rtol RTOL] [--atol ATOL] [--save-so PATH | --load-so PATH]
                        [--data-folder DATA_FOLDER | --shape-info SHAPE_INFO]
-                       [--compile-args COMPILE_ARGS]
-                       [--verify {onnxruntime,ref}] [--verify-all-ops]
-                       [--compile-using-input-shape] [--rtol RTOL]
-                       [--atol ATOL]
-                       model_path
-
-positional arguments:
-  model_path            Path to the ONNX model
 
 optional arguments:
   -h, --help            show this help message and exit
+  --model MODEL         Path to an ONNX model (.onnx or .mlir)
```

Comment:
This can actually now work with a` .mlir` file, we should mention this in the next stand-up